src/components/FloatingAction/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+import IconButton, { THEMES } from 'components/IconButton';
+
+export const VALID_FAB_THEMES = THEMES.filter((t) => t !== 'ghost');
+
+export interface FloatingActionProps {
+  /** Used to render a FDS Icon (should obnly be used for FDS Icons) */
+  icon: any;
```

Comment:
This should be uppercase. You're seeing a lowercase version (`icon`) because of this. The other version of icon you're seeing which is capitalized (`Icon`) is coming from the stories.jsx file in this same folder, will augment the props.

<img width="199" alt="Screen Shot 2021-12-09 at 3 50 56 PM" src="https://user-images.githubusercontent.com/708820/145481388-6052a972-c73f-499b-919f-46801f832c30.png">